locales/en.default.schema.json

```diff
@@ -992,10 +1007,12 @@
           "name": "Text",
           "settings": {
             "heading": {
-              "label": "Heading"
+              "label": "Heading",
+              "default": "Heading"
             },
             "subtext": {
-              "label": "Subtext"
+              "label": "Subtext",
+              "default": "<p>Share contact information, store details, and brand content with your customers.</p>"
```

Comment:
It's generally an antipattern to put HTML in translation files. This has burned me before a few times, like when a mobile native app reuses our locale files, or translators mis-interpreting what the HTML is doing.

Why do we need to do it here?